server/services/TransferService.spec.js

```diff
@@ -43,4 +45,49 @@ describe('TransferService', () => {
     expect(result).property('originating_wallet').eq('testName');
     expect(result).property('destination_wallet').eq('testName');
```

Comment:
While not very significant, it will be nice to have different name for `destination_wallet` in test.

Reply:
Yes, you are right, useless checking for that, should be changed.

Thanks, good suggestion, `withArgs`